src/main/java/com/orasi/bluesource/Accounts.java

```diff
@@ -55,10 +41,38 @@ public Accounts(OrasiDriver driver){
 	
 	/**Page Interactions**/
 
-	/*
+	/**
+	 * Checks if the rate provided matches the rate field on the Role page
+	 * @author David Grayson
+	 * @return <code>true</code> if the rate field from the Rates table on a Project page
+	 * matches the rate on the Rates table on the Role page, <code>false</code> if not.
+	 */
+	public boolean verifyRoleRate(String rate) {
```

Comment:
There isn't a hard standard on this, but I usually just do a getter type method, and then let the test class do any validations.  A good seperation between page class layer and test class later.  So in this method, I would just do the return:

`return driver.findWebtable(By.xpath("//th[contains(text(), 'Rate')]/../../..")).getRowWithCellText(rate, 5)`

Reply:
Hmm...on second review, I see you are seeing if the element with that rate value exists, i.e. that the row number is not 0....so that works.  You can discard my previous comment :)